test/e2e_tests/specs/Block/block.spec.ts

```diff
@@ -0,0 +1,344 @@
+        await userAPages.conversationList().clickUnblockConversation();
+        await userAModals.unblockWarningModal().clickUnblock();
+        // Step 4: User A send message to User B
+        await userAPages.conversationList().openConversation(userB.fullName);
+        await userAPages.conversation().sendMessage('Message after unblock');
+        // Step 5: User B receives message from User A
+        await userBPages.conversationList().openConversation(userA.fullName);
+        expect(await userBPages.conversation().messageCount()).toBe(1);
+        // Step 6: User B writes message to User A
+        await userAPages.conversation().sendMessage('Message after being unblocked');
+        // Step 7: User A receives message from User B
+        await userAPages.conversationList().openConversation(userB.fullName);
+        expect(await userAPages.conversation().messageCount()).toBe(2);
+      },
+    );
+
+    test(
+      'Verify you cannot add a person who blocked you to a group chat 0',
+      {tag: ['@TC-143', '@regression']},
+      async ({pageManager: userAPageManager, userBPageManager}) => {
+        const {pages: userAPages, modals: userAModals, components: userAComponents} = userAPageManager.webapp;
+        const conversationName = 'Groupchat with User A and User B';
+
+        // Step 1: User B blocks User A
+        await test.step('User B blocks User A', async () => {
+          await blockUserFromConversationList(userBPageManager, userA, {handleUnableToOpenModal: true});
+        });
+
+        // Step 2: User A wants to add B to a group chat after being blocked by User B
+        await test.step('Users A tries to add B to a group', async () => {
+          await userAComponents.conversationSidebar().isPageLoaded();
+          await userAPages.conversationList().clickCreateGroup();
+          await userAPages.groupCreation().setGroupName(conversationName);
+          await userAPages.startUI().selectUsers([userB.username]);
+          await userAPages.groupCreation().clickCreateGroupButton();
+
+          // Step 3: Modal 'modalConversationNotConnectedMessageOne' is visible
+          expect(userAModals.conversationNotConnected().isModalPresent()).toBeTruthy();
+        });
+      },
+    );
+
+    test.describe('User A and B are not in the same team and connect via UI', () => {
+      test.use({connectUsersViaAPI: false});
+
+      test(
+        'Verify you can block a user you sent a connection request from conversation list 0',
+        {tag: ['@TC-144', '@regression']},
+        async ({pageManager: userAPageManager, userBPageManager}) => {
+          const {pages: userBPages} = userBPageManager.webapp;
+
+          // Step 1: User A sends connection request to User B
+          await test.step('Preconditions: User A connects with User B', async () => {
+            await connectUsersManually(userA, userB, userAPageManager, userBPageManager, false);
```

Comment:
Are you sure this util is the right one to use here? To me the test title reads like the user should be blocked before the connection request is accepted, otherwise there wouldn't be a difference to previous tests or would it?